api-admin/build.gradle

```diff
@@ -14,18 +14,18 @@ apply from: 'publishing.gradle'
 
 dependencies {
 
-    implementation 'bio.terra:terra-common-lib'
+    implementation 'bio.terra:terra-common-lib:0.1.2-SNAPSHOT'
     implementation project(':core')
     implementation project(':populate')
     implementation 'org.apache.commons:commons-dbcp2'
     implementation 'org.apache.commons:commons-text:1.10.0'
     implementation 'commons-beanutils:commons-beanutils:1.9.4'
-    implementation 'org.springframework.boot:spring-boot-starter:2.7.10'
-    implementation 'org.springframework.boot:spring-boot-starter-data-jdbc:2.7.10'
-    implementation 'org.springframework.boot:spring-boot-starter-web:2.7.10'
+    implementation 'org.springframework.boot:spring-boot-starter:3.1.2'
```

Comment:
Upgrading to 3.1.2 right now because TCL doesn't work with 3.1.5 yet. TCL logging utilities use some deprecated code from Logback which was removed in the version of Logback that comes along with Spring Boot 3.1.5.